### PR TITLE
Initial release notes

### PR DESCRIPTION
**Pull request by esbenbach**
The changelog for release 96.

### CODE REVIEW

File: source/changelog/r96.inc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.. _changelog-r96:
+
+****************************
+Release December 12th 2021
+****************************
+The last release of the year 2021. This brings us to a total of 13 release for 2021.
```
Comment (BrynjarS) [inline source/changelog/r96.inc:6]: Spelling: "13 release" -> "13 releases"

File: source/changelog/r96.inc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+The release was rolled out entirely on the 12th of December, all API, backend and tenant clients have been rolled out at roughly the same time.
+Without further ado, lets head on to the interesting bits.
```
Comment (BrynjarS) [inline source/changelog/r96.inc:8]: Spelling: lets -> let's

File: source/changelog/r96.inc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+API and Backend
+===============
+The following new features and improvements are now available in the API and the backend.
+
+* Added support for importing existing Vipps Recurring agreements.
+* Added support for getting the remote Vipps Agreement details based on an existing agreement.
+* Added support for skipping the Vipps Recrruing Landing page (requires approval/setting by Vipps).
```
Comment (BrynjarS) [inline source/changelog/r96.inc:17]: Spelling: Recrruing

File: source/changelog/r96.inc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* Minor documentation adjustments.
+* When Subscriber Contacts are deleted, the subscription using the contact (Invoice Address) will automatically revert to the subscribers primary contact.
+* Scheduled Plan Changes `At Renewal` will now override any existing changes of the same type and return the correct change id.
+* Fixed an issue where AvtaleGiro Payment Demands would be calculated with a wrong Due Date due to an offset between the UTC core and the Norwegian timezone deadlines of Mastercard Payment Services.
```
Comment (BrynjarS) [inline source/changelog/r96.inc:21]: Norwegian timezone -> CET?

Comment (esbenbach) [reply source/changelog/r96.inc:21]: Nope, its specifically Norwegian because CET and CEST are both Norwegian ;)

**Pull request merged**